src/server/rpc/storage.rs

```diff
@@ -1717,13 +1845,18 @@ mod tests {
             predicate: make_state_ma_predicate(),
             group_keys: vec![String::from("tag1")],
             group,
-            aggregate: None,
+            aggregate: Some(RPCAggregate {
+                r#type: AggregateType::Sum as i32,
+            }),
             hints: 0,
         };
 
         let expected_request = QueryGroupsRequest {
             predicate: "Predicate { exprs: [#state Eq Utf8(\"MA\")] range: TimestampRange { start: 150, end: 200 }}".into(),
-            group_columns: vec![String::from("tag1")],
+            gby_agg : GroupByAndAggregate::Columns {
```

Comment:
```suggestion
            gby_agg: GroupByAndAggregate::Columns {
```

I would have expected rustfmt to catch this, hmm